src/pages/home/report/ReportActionItem.js

```diff
@@ -70,59 +67,45 @@ const propTypes = {
 
 const defaultProps = {
     draftMessage: '',
-    hasOutstandingIOU: false,
 };
 
-class ReportActionItem extends Component {
-    constructor(props) {
-        super(props);
-        this.popoverAnchor = undefined;
-        this.state = {
-            isContextMenuActive: ReportActionContextMenu.isActiveReportAction(props.action.reportActionID),
-        };
-        this.checkIfContextMenuActive = this.checkIfContextMenuActive.bind(this);
-        this.showPopover = this.showPopover.bind(this);
-        this.renderItemContent = this.renderItemContent.bind(this);
-        this.toggleReaction = this.toggleReaction.bind(this);
-    }
+function ReportActionItem(props) {
+    const [isContextMenuActive, setIsContextMenuActive] = React.useState(ReportActionContextMenu.isActiveReportAction(props.action.reportActionID));
+    const previousDraftMessage = React.useRef('');
+    const textInput = React.useRef();
```

Comment:
I think a good convention would be to name these `*Ref` e.g. `textInputRef`, `previousDraftMessageRef`